paper.tex

```diff
@@ -69,6 +69,8 @@ \subsection{Detector models}
 
 \subsubsection{Image detector models}
 
+The \textbf{AIorNot} detector by Nahrawy on Hugging Face is a pretrained swin-tiny-patch4-window7-224 finetuned on the AI or Not dataset. swin-tiny-patch4-window7-224 by microsoft is a new vision Transformer \cite{https://arxiv.org/pdf/2103.14030.pdf}.
```

Comment:
Here is a suggested addition. Additionally we need to fix the citation here.
```suggestion
The \textbf{AIorNot} detector by Nahrawy on Hugging Face is a pretrained swin-tiny-patch4-window7-224 finetuned on the AI or Not dataset. Swin-Transformer by Microsoft is a new Vision Transformer which makes use of hierarchical windows. This reduces the self-attention to not-overlapping windows of other areas of the IMAGE. \cite{https://arxiv.org/pdf/2103.14030.pdf}.
```
I would not wait on the other PR, but rather merge them sooner, then later to get a full text soon, which we then can rephrase if necessary.

Reply:
Don't like the two 'Transfomer' in one sentence so close together

Reply:
its 2 names, cant romove one transformer/replace no?